src/ophyd_async/fastcs/panda/_trigger.py

```diff
@@ -51,6 +60,93 @@ async def stop(self):
         await wait_for_value(self.seq.active, False, timeout=1)
 
 
+class ScanSpecSeqTableTriggerLogic(FlyerController[ScanSpecInfo]):
+    def __init__(self, seq: SeqBlock, name="") -> None:
+        self.seq = seq
+        self.name = name
+
+    async def prepare(self, value: ScanSpecInfo):
+        await self.seq.enable.set(PandaBitMux.ZERO)
+        slice = Path(value.spec.calculate()).consume()
+
+        gaps = self._calculate_gaps(slice)
+        if gaps[0] == 0:
+            gaps = np.delete(gaps, 0)
+        scan_size = len(slice)
+
+        gaps = np.append(gaps, scan_size)
+        fast_axis = slice.axes()[len(slice.axes()) - 2]
```

Comment:
I believe the code originally was using `len(slice.axes()) - 2` because at the time the last axis would always be the `DURATION`. As we're not using that anymore I believe it can be done like this:
```suggestion
        fast_axis = slice.axes()[len(slice.axes()) - 1]
```